packages/odyssey-storybook/.storybook/components/MuiThemeDecorator.tsx

```diff
@@ -15,18 +15,22 @@ const styles = {
 
 const odysseyTheme = createOdysseyMuiTheme({ odysseyTokens });
 
-export const MuiThemeDecorator: Decorator = (Story, context) => (
-  <OdysseyThemeProvider>
-    <OdysseyTranslationProvider languageCode={context.globals.locale}>
-      {/* @ts-expect-error type mismatch on "typography" */}
-      <StorybookThemeProvider theme={odysseyTheme}>
-        <CssBaseline />
-        <div style={styles}>
-          <ScopedCssBaseline>
-            <Story />
-          </ScopedCssBaseline>
-        </div>
-      </StorybookThemeProvider>
-    </OdysseyTranslationProvider>
-  </OdysseyThemeProvider>
-);
+export const MuiThemeDecorator: Decorator = (Story, context) => {
+  const { canvasElement } = context;
+  const shadowRootElement = canvasElement.parentElement ?? undefined;
+  return (
+    <OdysseyProvider shadowDomElement={shadowRootElement}>
```

Comment:
You should be able to pass `languageCode` directly to `OdysseyProvider` here. And be able to remove `OdysseyTranslationProvider`
```suggestion
    <OdysseyProvider shadowDomElement={shadowRootElement} languageCode={context.globals.locale}>
```

Reply:
Nice! I'll make the change